### PR TITLE
Improving Login menu language strings

### PR DESCRIPTION
**Pull request by infograf768**
As title says. Changed Custom URL to Internal URL and added a placeholder.

After patch:
screen shot 2016-05-08 at 19 24 43

### CODE REVIEW

File: administrator/language/en-GB/en-GB.com_users.ini [hunk 1]
```diff
@@ -93,12 +93,13 @@ COM_USERS_ERROR_SECRET_CODE_WITHOUT_TFA="You have entered a Secret Code but two
 COM_USERS_FIELD_CATEGORY_ID_LABEL="Category"
 COM_USERS_FIELD_ID_LABEL="ID"
 COM_USERS_FIELD_LOGIN_MENUITEM="Menu Item"
-COM_USERS_FIELD_LOGIN_REDIRECT_CHOICE_DESC="'Custom URL' lets you manually enter any internal url in the Redirect field. 'Menu Item' lets you directly select an existing menu item.<br />For a multilingual site, it is advised to use 'Menu Item'."
+COM_USERS_FIELD_LOGIN_REDIRECT_PLACEHOLDER="index.php?Itemid=999&lang=en-GB"
+COM_USERS_FIELD_LOGIN_REDIRECT_CHOICE_DESC="'Internal URL' lets you manually enter any internal url in the Redirect field. 'Menu Item' lets you directly select an existing menu item.<br />For a multilingual site, it is advised to use 'Menu Item'."
```
Comment (brianteeman) [inline administrator/language/en-GB/en-GB.com_users.ini]: can you change "advised" to "recommended"

File: administrator/language/en-GB/en-GB.ini [hunk 2]
```diff
@@ -196,11 +196,11 @@ JFIELD_LANGUAGE_DESC="Assign a language to this article."
 JFIELD_LANGUAGE_LABEL="Language"
 JFIELD_LOGIN_IMAGE_DESC="Select or upload an image to display on login page."
 JFIELD_LOGIN_IMAGE_LABEL="Login Image"
-JFIELD_LOGIN_REDIRECT_URL_DESC="If an URL is entered here, users will be redirected to it after login. The URL must not be an external one."
+JFIELD_LOGIN_REDIRECT_URL_DESC="If an URL is entered here, users will be redirected to it after login.<br />The URL must be internal (ex: index.php?Itemid=999)."
```
Comment (brianteeman) [inline administrator/language/en-GB/en-GB.ini]: Can you change this to
"If a URL.."

Yes it is a quirk of english and breaks the rule you know but it is correct

File: administrator/language/en-GB/en-GB.ini [hunk 3]
```diff
@@ -196,11 +196,11 @@ JFIELD_LANGUAGE_DESC="Assign a language to this article."
...
 JFIELD_LOGIN_REDIRECT_URL_LABEL="Login Redirect"
 JFIELD_LOGOUT_IMAGE_DESC="Select or upload an image to display on logout page."
 JFIELD_LOGOUT_IMAGE_LABEL="Logout Image"
-JFIELD_LOGOUT_REDIRECT_URL_DESC="If an URL is entered here, users will be redirected to it after logout. The URL must not be an external one."
+JFIELD_LOGOUT_REDIRECT_URL_DESC="If an URL is entered here, users will be redirected to it after logout.<br />The URL must be internal (ex: index.php?Itemid=999)."
```
Comment (brianteeman) [inline administrator/language/en-GB/en-GB.ini]: Can you change this to
"If a URL.."

Yes it is a quirk of english and breaks the rule you know but it is correct

**Comment by brianteeman** [comment]
Made some comments inline

**Comment by andrepereiradasilva** [comment]
it seems travis is still recovering from today merge frenzy.

**Comment by infograf768** [comment]
@brianteeman 
Done.

File: administrator/language/en-GB/en-GB.ini [hunk 4]
```diff
@@ -196,11 +196,11 @@ JFIELD_LANGUAGE_DESC="Assign a language to this article."
 JFIELD_LANGUAGE_LABEL="Language"
 JFIELD_LOGIN_IMAGE_DESC="Select or upload an image to display on login page."
 JFIELD_LOGIN_IMAGE_LABEL="Login Image"
-JFIELD_LOGIN_REDIRECT_URL_DESC="If an URL is entered here, users will be redirected to it after login. The URL must not be an external one."
+JFIELD_LOGIN_REDIRECT_URL_DESC="If a URL is entered here, users will be redirected to it after login.<br />The URL must be internal (ex: index.php?Itemid=999)."
```
Comment (brianteeman) [inline administrator/language/en-GB/en-GB.ini]: Sorry one last change
ex should be eg 

No periods just as this
 internal (eg: index.php?Itemid=999)."

File: administrator/language/en-GB/en-GB.ini [hunk 5]
```diff
@@ -196,11 +196,11 @@ JFIELD_LANGUAGE_DESC="Assign a language to this article."
...
 JFIELD_LOGIN_REDIRECT_URL_LABEL="Login Redirect"
 JFIELD_LOGOUT_IMAGE_DESC="Select or upload an image to display on logout page."
 JFIELD_LOGOUT_IMAGE_LABEL="Logout Image"
-JFIELD_LOGOUT_REDIRECT_URL_DESC="If an URL is entered here, users will be redirected to it after logout. The URL must not be an external one."
+JFIELD_LOGOUT_REDIRECT_URL_DESC="If a URL is entered here, users will be redirected to it after logout.<br />The URL must be internal (ex: index.php?Itemid=999)."
```
Comment (brianteeman) [inline administrator/language/en-GB/en-GB.ini]: Sorry one last change
ex should be eg 

No periods just as this
 internal (eg: index.php?Itemid=999)."

**Comment by brianteeman** [comment]
Sorry one last change and then we can make it RTC

**Comment by brianteeman** [comment]
RTC<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/10336">issues.joomla.org/joomla-cms/10336</a>.</sub>

**Comment by MATsxm** [comment]
I have tested this item :white_check_mark: successfully on b09ce729f917d2dbc4495192333815d54741e8c5<br /><br />Thanks<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/10336">issues.joomla.org/joomla-cms/10336</a>.</sub>

**Comment by zero-24** [comment]
please remove the label @joomla-cms-bot

**Pull request merged**